11-Backbonejs/02-Backbone-Views.md

```diff
@@ -85,25 +78,27 @@ Rails views are awesome!  They are written in an HTMLish format with special han
 
 ### Underscorejs
```

Comment:
If we're not going to use the "js" suffix for Backbone, we should also not use it for Underscore.